data/strings/strings.txt

```diff
@@ -4997,49 +4997,50 @@
     zh-Hant = OpenStreetMap 數據：%@
 
   [are_you_sure]
-    comment = Confirmation in downloading countries dialog
+    comment = Confirmation for OpenStreetMap log out.
     tags = android
-    en = Are you sure you want to continue?
-    af = Is u seker u wil voortgaan?
-    ar = هل أنت واثق أنك تريد الاستمرار؟
-    az = Davam etmək istədiyinizə əminsiniz?
-    be = Вы сапраўды хочаце працягнуць?
-    bg = Сигурни ли сте, че искате да продължите?
-    ca = Segur que voleu continuar?
-    cs = Opravdu chcete pokračovat?
-    da = Er du sikker på, at du vil fortsætte?
-    de = Möchten Sie wirklich fortfahren?
-    el = Θέλετε σίγουρα να συνεχίσετε?
-    es = ¿Seguro que desea continuar?
-    et = Oled kindel, et soovid jätkata?
-    eu = Ziur jarraitu nahi duzula?
-    fa = برای ادامه کار مطمئن اید؟
-    fi = Oletko varma että haluat jatkaa?
-    fr = Êtes-vous certain de vouloir continuer ?
-    he = האם הנך בטוח שברצונך להמשיך?
-    hi = क्या आप वाकई जारी रखना चाहते हैं?
-    hu = Valóban folytatni akarja?
-    id = Apakah Anda yakin ingin melanjutkan?
-    it = Sei sicuro di voler continuare?
-    ja = 続行しますか？
-    ko = 계속하겠습니까?
-    lt = Tikrai norite tęsti?
-    mr = खात्रीपूर्वक पुढे जायचे?
-    nb = Er du sikker på at du ønsker å fortsette?
-    nl = Weet u zeker dat u wilt doorgaan?
-    pl = Kontynuować?
-    pt = Quer mesmo continuar?
-    pt-BR = Tem certeza que deseja continuar?
-    ro = Vrei să continui?
-    ru = Вы уверены, что хотите продолжить?
-    sk = Naozaj chcete pokračovať?
-    sv = Är du säker på att du vill fortsätta?
-    th = คุณแน่ใจหรือไม่ว่าคุณต้องการดำเนินการต่อ?
-    tr = Devam etmek istediğinizden emin misiniz?
-    uk = Продовжити?
-    vi = Bạn có chắc muốn tiếp tục không?
-    zh-Hans = 您确定要继续吗？
-    zh-Hant = 您確定要繼續嗎？
+    en = Are you sure you want to log out of your OpenStreetMap account?
+    af = Is jy seker jy wil by jou OpenStreetMap-rekening afmeld?
+    ar = هل أنت متأكد أنك تريد تسجيل الخروج من حساب OpenStreetMap الخاص بك؟
+    az = OpenStreetMap hesabınızdan çıxmaq istədiyinizə əminsiniz?
+    be = Вы ўпэўнены, што хочаце выйсці са свайго ўліковага запісу OpenStreetMap?
+    bg = Сигурни ли сте, че искате да излезете от профила си в OpenStreetMap?
+    ca = Esteu segur que voleu tancar la sessió del vostre compte d'OpenStreetMap?
+    cs = Opravdu se chcete odhlásit ze svého účtu OpenStreetMap?
+    da = Er du sikker på, at du vil logge ud af din OpenStreetMap-konto?
+    de = Sind Sie sicher, dass Sie sich von Ihrem OpenStreetMap-Konto abmelden möchten?
+    el = Είστε σίγουροι ότι θέλετε να αποσυνδεθείτε από το λογαριασμό σας στο OpenStreetMap;
+    es = ¿Está seguro de que desea salir de su cuenta de OpenStreetMap?
```

Comment:
Shorter and gender neutral alternatives "¿Quiere salir de su cuenta de OpenStreetMap?" or "¿Desea salir de su cuenta de OpenStreetMap?"